File: ParkingPlace/app/src/main/java/com/rmj/parking_place/dto/ParkingPlacesInitialDTO.java
package com.rmj.parking_place.dto;

import com.rmj.parking_place.model.ParkingPlace;

import java.util.List;

public class ParkingPlacesInitialDTO {
    private Long zoneId;
    private Long version;
    private List<ParkingPlace> parkingPlaces;


    public ParkingPlacesInitialDTO() {

    }

    public Long getZoneId() {
        return zoneId;
    }

    public void setZoneId(Long zoneId) {
        this.zoneId = zoneId;
    }

    public Long getVersion() {
        return version;
    }

    public void setVersion(Long version) {
        this.version = version;
    }

    public List<ParkingPlace> getParkingPlaces() {
        return parkingPlaces;
    }

    public void setParkingPlaces(List<ParkingPlace> parkingPlaces) {
        this.parkingPlaces = parkingPlaces;
    }
}

File: ParkingPlaceServer/ParkingPlaceServer/DTO/ParkingPlacesInitialDTO.cs
using Newtonsoft.Json;
using ParkingPlaceServer.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ParkingPlaceServer.DTO
{
	
	public class ParkingPlacesInitialDTO
	{

		[JsonProperty("zoneId")]
		public long ZoneId { get; set; }


		[JsonProperty("version")]
		public long Version { get; set; }


		[JsonProperty("parkingPlaces")]
		public List<ParkingPlace> ParkingPlaces { get; set; }


		public ParkingPlacesInitialDTO()
		{

		}

		public ParkingPlacesInitialDTO(long zoneId, long version, List<ParkingPlace> parkingPlaces)
		{
			ZoneId = zoneId;
			Version = version;
			ParkingPlaces = parkingPlaces;
		}
	}
}